src/Controllers/Common/ContentController.cs

```diff
@@ -809,10 +823,10 @@ public IActionResult GetAvatarByUserId([FromForm] Guid userId)
     [Produces("application/xml")]
     [Route("ContentWebService.asmx/GetActiveRaisedPet")] // used by World Of Jumpstart
     [VikingSession(UseLock=false)]
-    public RaisedPetData[] GetActiveRaisedPet(Viking viking, [FromForm] string userId, [FromForm] int petTypeID) {
+    public RaisedPetData[] GetActiveRaisedPet(Viking viking, [FromForm] string? apiKey, [FromForm] string userId, [FromForm] int petTypeID) {
```

Comment:
apiKey is send always (if I'm not mistaken) ... so just `string` instead of `string?` ... and the sodoff convention is to add it as the last argument (same for `GetUnselectedPetByTypes`)